website/content/sdk/guides/intros/stocks/government.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Government
+---
+
+The Government module provides programmatic access to the same functions from the [OpenBB Terminal](https://docs.openbb.co/terminal/guides/intros/stocks/gov). They provide an excellent supplement to fundamental and macroeconomic research. Please note that this module will only return data from the USA.
+
+## How to Use
+
+Below is a list of all the commands and a brief description. There are two types of function: ticker-specific, and non-specific. Many of these tables also have a companion `_chart` function.
+
+|Path |Type| Description |
+|:----------------|:------:|----------------------------:|
+|openbb.stocks.gov.contracts|Ticker |Contracts Awarded to a Company |
+|openbb.stocks.gov.gtrades |Ticker |Reported Trades in a Company's Stock by Members of the US Congress and Senate |
+|openbb.stocks.gov.histcont|Ticker |The Quarterly Total Amount Paid to a Company by the Government |
+|openbb.stocks.gov.lastcontracts |All |The Latest Invoices Paid by the US Treasury Department |
+|openbb.stocks.gov.lasttrades| All|The Latest Reported Trades Made by Members of the US Congress and Senate |
+|openbb.stocks.gov.lobbying|Ticker | Recorded Lobbying Efforts by a Company |
+|openbb.stocks.gov.qtrcontracts|All |Ranking Companies by Total Amount Rewarded |
+|openbb.stocks.gov.topbuys|All |The Top Buyers in Office |
+|openbb.stocks.gov.toplobbying|All |Corporate Lobbyist Activity and the Specific Issues |
+|openbb.stocks.gov.topsells|All |The Top Sellers in Office |
+
+Alternatively, Python's built-in help will display the contents of the module, or specific function.
+
+```python
+help(openbb.stocks.gov)
+```
+
+## Examples
+
+Let's import the SDK in a way that reduces the amount of typing required to access the module.
+
+### Import Statements
+
+```python
+from openbb_terminal.sdk import openbb
+gov = openbb.stocks.gov
+```
+
+The examples below will assume that the block above is included at the top of the file.
+
+### Toplobbying
+
+This command shows who is spending money in Washington and what their intentions are.
+
+```python
+gov.toplobbying()
+```
+
+|    | Date       | Ticker   | Client                          |   Amount | Issue                | Specific_Issue  |
```

Comment:
Please shorten this table, it is currently very large. I'd suggest doing `limit=3` for example. People get the gist of it with a few inputs.

Reply:
done